### ISSUE TITLE
NMEA2000-out: NMEA 2000 Vendor name

### ISSUE THREAD

**Issue by DevoDave** [victronenergy/venus#854]
NMEA2000 labeling of the Venus product and devices is not accurate.

Vendor comes up as "1.0" for my CerboGX and the devices come up as unknown-n2K-device (or similar.. working from memory.)

Should hopefully be simple to fix and just require some labels added?

**Comment by mpvader** [comment]
Hi @DevoDave , thanks for reporting. Please tell me what software you use in which you see this information?

**Comment by DevoDave** [comment]
Raymarine Lighthouse 3 most current version.

**Comment by virtuvas** [comment]
odd, on my setup with around 20 N2K devices all are fine except for the Model which comes up as 1.0. Both  manufacturer and name are spot on!

That is a raspberry pi 3B+, a BMV700 and a Bluesolar MPPT 30/100
I'm running the large built with SignalK (excuse the salt deposits on the Garmin, came back 3days ago and haven't cleaned up yet)

![image](https://user-images.githubusercontent.com/17377407/135489167-da830a7e-08b9-4f9e-82f7-3df9a8debdf0.png)

![image](https://user-images.githubusercontent.com/17377407/135489394-57b10eb8-34fa-4f27-844b-e7c7cd2528ef.png)

![image](https://user-images.githubusercontent.com/17377407/135489435-27eeef33-be0d-468c-8fb9-3d6b85ead2ff.png)

![image](https://user-images.githubusercontent.com/17377407/135489620-be7896f5-38ab-4c87-be9f-203548a69680.png)

![image](https://user-images.githubusercontent.com/17377407/135489722-5ab07505-f3b4-4959-890a-caab47aa5967.png)

**Comment by mpvader** [comment]
hi both, this is just a note that I've seen it. Sorry for not responding earlier, and thanks for reporting!

I have to warn, it might take a while before we look into this.

**Comment by martinbosma** [comment]
Have checked this with Simrad, Garmin and Raymarine MFDs and also with Actisense NMEAReader and Maretron N2KAnalyzer and it looks like everything is according to NMEA2000 specification. I did see some "unknown-n2K-device" lables on the Raymarine, but this appears to be a (display) issue on their side, not on our side.

I did notice that most MFD's show the "Manufacturer Model version" field as "Model", while it actually is a hardware version, hence the (fixed) 1.0. The "Manufacturer Model ID" contains the model name, e.g. "BMV-700". All tested MFD's displayed this correctly.

All seems to be working as expected, so closing this issue.

![2022-05-20 11_22_26-NMEA Reader -  COM9_ Actisense NGT](https://user-images.githubusercontent.com/6614648/169500104-58cf721a-31aa-4a38-9d6b-e35b8694867d.png)
![2022-05-20 11_21_57-NMEA Reader -  COM9_ Actisense NGT](https://user-images.githubusercontent.com/6614648/169500197-a9d0fa2f-e47e-4dda-988a-0624525b4789.png)
![2022-05-20 11_29_17-Maretron N2KAnalyzer V3, Version 3 2 0 2 - VesselName](https://user-images.githubusercontent.com/6614648/169500257-52d430b3-1c16-4c9e-9cde-7a2cf087ed26.png)
![2022-05-20 11_29_41-Properties_ (0xE0) BMV-700 - 0000508](https://user-images.githubusercontent.com/6614648/169500269-e80bfbed-a444-44ac-956b-fd8ce7383178.png)
